src/Sarif.WorkItems/SarifWorkItemsExtensions.cs

```diff
@@ -12,6 +12,11 @@ public static class SarifWorkItemsExtensions
     {
         public static bool ShouldBeFiled(this Result result)
         {
+            if (result == null)
+            {
+                throw new ArgumentNullException(nameof(result));
+            }
+
```

Comment:
I suggest removing this, because it is additional work and suggests a general pattern we should adhere that doesn't seem to have good ROI.